Blogpost1.php

```diff
@@ -0,0 +1,28 @@
+<html>
+	<head>
+		<title>May 17th blog</title>
+
+		<link rel='stylesheet' href="blogpost.css">
+
+		<link href='https://fonts.googleapis.com/css?family=Barrio' rel='stylesheet'>
+
+		<link href='https://fonts.googleapis.com/css?family=Delius Unicase' rel='stylesheet'>
```

Comment:
It should be possible to include both of the fonts in a single `<link>` tag instead of using two. This will make your site a bit faster because it'll only need to make one request to Google instead of two.